src/tpl/CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ if (NOT TARGET fmt::fmt)
       set(fmt_DIR ${CMAKE_INSTALL_PREFIX} CACHE PATH "")
       set(FMT_INSTALL ON)
       set(FMT_SYSTEM_HEADERS ON)
+      set(UMPIRE_FMT_TARGET fmt::fmt-header-only CACHE STRING "" FORCE)
```

Comment:
Why is this different than the default in SetupUmpireOptions.cmake? Both should be the same to avoid confusion.

Reply:
This restores the previous behavior (where the submodule was header only)

Reply:
Per discussion, the default is now fmt::fmt-header-only everywhere, not just when using the submodule.